### PR TITLE
Layout project transitions

### PR DESCRIPTION
**Pull request by mar7in91Wolox**
# **Summary**

- I added the borders for each tech box.
- I added the animations.

# **Trello Card**

https://trello.com/c/LR65MJ4f

# **Screeshots**

image

transitions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,78 +11,96 @@
...
+
     <h1 class="title-text">Front-End Tools</h1>
 
-    <div class="tech">
-      <img class="tech-logo" alt="React icon" src="img/react_icon.svg">
-      <h3 class="tech-title">ReactJS</h3>
-      <h3 class="tech-release">Release: 2013</h3>
-      <p class="tech-info">React is a declarative, component-based Javascript library for building user interfaces.</p>
+    <div  class='tech-border'>
+      <div class="tech">
```
Comment (mcallegari10) [inline index.html]: Remember that you should avoid wrapping single elements into a `div`. For this feature, you can use a pseudo-element too! 😸

**Comment by mar7in91Wolox** [comment]
All solved!

File: scss/tech.scss [hunk 2]
```diff
@@ -1,9 +1,16 @@
+body {
+  display: flex;
+  flex-direction: column;
```
Comment (mcallegari10) [inline scss/tech.scss]: Why do you need this on the body tag?

File: scss/tech.scss [hunk 3]
```diff
@@ -12,11 +19,29 @@
...
+    height: 100%;
+    left: -10px;
+    position: absolute;
+    width: 100%;
+    transition: top 0.5s, left 0.5s;
+    top: -10px;    
+  }  
+}
+
+.tech:hover::before {
```
Comment (mcallegari10) [inline scss/tech.scss]: You can do this using the nesting feature of SASS:

```scss
.tech {

  &::before {
  }

  &:hover {

     &::before {
 
...
```

File: scss/tech.scss [hunk 4]
```diff
@@ -25,15 +50,15 @@
 }
 
 .tech-title {
-  font-size: $text-medium;
+  margin-top: 10px;
```
Comment (mcallegari10) [inline scss/tech.scss]: Is this really necessary? I think you were using the `justify-content: space-between` to separate the elements of the tech. BTW, the `font-size` was OK

File: scss/tech.scss [hunk 5]
```diff
@@ -25,15 +50,15 @@
...
 }
 
 .tech-release {
   font-weight: bold;
-  font-size: $text-medium;
+  margin-top: 10px;
```
Comment (mcallegari10) [inline scss/tech.scss]: Same here

File: scss/tech.scss [hunk 6]
```diff
@@ -25,15 +50,15 @@
...
 }
 
 .tech-info {
-  font-size: $text-small;
+  margin-top: 15px;
```
Comment (mcallegari10) [inline scss/tech.scss]: And here

File: scss/tech.scss (hunk 2 above)
Comment (mar7in91Wolox) [reply scss/tech.scss]: It was a suggestions from Gabo. Should I do a class for body tag?

File: scss/tech.scss (hunk 3 above)
Comment (mar7in91Wolox) [reply scss/tech.scss]: Done!

File: scss/tech.scss (hunk 4 above)
Comment (mar7in91Wolox) [reply scss/tech.scss]: It was a suggestion from Gabo.

File: scss/tech.scss (hunk 5 above)
Comment (mar7in91Wolox) [reply scss/tech.scss]: It was a suggestion from Gabo.

File: scss/tech.scss (hunk 6 above)
Comment (mar7in91Wolox) [reply scss/tech.scss]: It was a suggestion from Gabo.

File: scss/tech.scss (hunk 2 above)
Comment (mcallegari10) [reply scss/tech.scss]: No need, this is OK but I just wanted to know why 😝

**Pull request merged**